src/app/layout.tsx

```diff
@@ -1,23 +1,28 @@
-import type { Metadata } from 'next';
-import './globals.css';
+'use client';
 
-export const metadata: Metadata = {
-  title: 'Create Next App',
-  description: 'Generated by create next app',
-};
+import Form from '@/components/Form';
+import { useRouter } from 'next/navigation';
+import './globals.css';
 
 export default function RootLayout({
   children,
 }: Readonly<{
   children: React.ReactNode;
 }>) {
+  const router = useRouter();
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    router.push('/start');
```

Comment:
[Q]
ここは result ですかね？
```suggestion
    router.push('/result');
```